pysb/tests/test_exporters.py

```diff
@@ -112,7 +114,12 @@ def check_convert(model, format):
             # Check network generation and single-step integrator
```

Comment:
As you pointed out in your previous comment, we actually just need to evaluate the RHS once which is different from stepping the integrator. This comment would be more accurate as something like "Check network generation and force RHS evaluation".

Reply:
Comment fixed as requested